template/client.tmpl

```diff
@@ -20,8 +23,17 @@ func Get{{ .CamelCaseName }}Client() ({{ .CamelCaseName }}Client, error) {
 		return Client, nil
 	}
 
-	logrus.Info("Creating {{ .Name }} gRPC client")
-	conn, err := grpc.Dial("{{ .SnakeCaseName }}:80", grpc.DialOption(grpc.WithInsecure()))
+	logrus.Infof("Creating %s gRPC client", serviceName)
+
+    rc, _ := registry.NewConsulClient("localhost:8500")
```

Comment:
People may run Consul on a different URL, so I'd check for an ENV var or similar

Reply:
true, somehow my edit got lost? it was supposed to look like this

```
logrus.Infof("Creating %s gRPC client", serviceName)
    registryAddress := os.Getenv("REGISTRY_ADDRESS")
    if registryAddress == "" {
        registryAddress = "localhost:8500"
    }
    rc, _ := registry.NewConsulClient(registryAddress)
    entries, _, err := rc.Service(serviceName, "", nil)
	if err != nil {
		logrus.Fatalf("Failed to find service '%s'. error: %v", serviceName, err)
		os.Exit(1)
	}
	srv := entries[0]
	s := fmt.Sprintf("%s:%d", srv.Service.Address, srv.Service.Port)
	conn, err := grpc.Dial(s, grpc.DialOption(grpc.WithInsecure()))
```

Reply:
Yet again, after the abstracted registry, this will also have to change